Tests/js/views/fields/assets/ez-relationlist-editview-tests.js

```diff
@@ -0,0 +1,569 @@
+/*
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+YUI.add('ez-relationlist-editview-tests', function (Y) {
+    var viewTest, registerTest, universalDiscoveryRelationTest, getFieldTest, getEmptyFieldTest;
+
+    viewTest = new Y.Test.Case({
+        name: "eZ Relation list View test",
+
+        _getFieldDefinition: function (required) {
+            return {
+                isRequired: required
+            };
+        },
+
+        setUp: function () {
+
```

Comment:
pretty sure this method can be stripped down, for instance is it necessary to mock the `get` methods of the destination contents ?